packages/react-kit/src/components/DateInput/DateInput.tsx

```diff
@@ -433,6 +433,10 @@ class RawDateInput extends React.Component<TDateInputFullProps, TDateInputState>
 	};
 
 	onSteppableInputClick = (e: React.MouseEvent<HTMLElement>) => {
+		if (this.props.isDisabled) {
```

Comment:
Why here and not in `SteppableInput`?

Reply:
Ok, I moved condition into SteppableInput

Reply:
The same applies to `SteppableInput`

Reply:
Ok! Now I moved it to Input component